### PR TITLE
[ch202] As a user, I can see background image in Sign-up screen

### PR DESCRIPTION
**Pull request by Raksani**
https://app.clubhouse.io/cobda/story/202/as-a-user-i-can-see-background-image-in-sign-up-screen

## Why?

To apply the background image design on the signup page

## Changes

Having background image on the signup page

## Screenshot(s)

**Before**
Screenshot 2564-03-10 at 23 44 35

**After**
Screenshot 2564-03-10 at 23 43 18

### CODE REVIEW

File: src/styles/layouts/_layout-authentication.scss [hunk 1]
```diff
@@ -36,6 +41,13 @@
 
     @include respond-above('md') {
       padding: 4.5% 6%;
+
+      background: {
+        image: url('/images/sign-up-background.jpg');
+        repeat: no-repeat;
+        size: cover;
+        position: top 10% center;
```
Comment (LilBank) [inline src/styles/layouts/_layout-authentication.scss:44]: No need for `top`.
```suggestion
        position: center;
```

File: src/styles/layouts/_layout-authentication.scss [hunk 2]
```diff
@@ -18,7 +18,12 @@
   }
 
   @include respond-above('xl') {
+    grid-template-columns: 1fr minmax(rem(1104px), rem(1440px)) 1fr;
```
Comment (LilBank) [inline src/styles/layouts/_layout-authentication.scss]: Currently the background image is limited to `width` of `1440px`. It should cover all the `main`. Try using mobile `grid-template-areas` for every breakpoint and set the **max** of the `form` at `sign-in-form`.

**Review by LilBank — CHANGES_REQUESTED** [review]

File: src/styles/layouts/_layout-authentication.scss (hunk 1 above)
Comment (Raksani) [reply src/styles/layouts/_layout-authentication.scss:44]: I think it's more similar to the design when the top goes down. What do you think? 

design
Screenshot 2564-03-12 at 01 29 57

center with top 10%
Screenshot 2564-03-12 at 01 33 41

center
Screenshot 2564-03-12 at 01 33 06

Comment (LilBank) [reply src/styles/layouts/_layout-authentication.scss:44]: It's fine for me, what do others think?

**Review by kimvcp — APPROVED** [review]

**Review by kimvcp — APPROVED** [review]

**Review by LilBank — APPROVED** [review]

**Review by Kittinske15 — APPROVED** [review]

**Pull request merged**